lua/claudecode/init.lua

```diff
@@ -323,6 +323,40 @@ function M._create_commands()
       "Terminal module not found. Terminal commands (ClaudeCode, ClaudeCodeOpen, ClaudeCodeClose) not registered."
     )
   end
+
+  vim.api.nvim_create_user_command("ClaudeSelectModel", function()
+    M.open_with_model()
+  end, {
+    desc = "Select and open Claude terminal with chosen model",
+  })
+end
+
+M.open_with_model = function()
+  local models = {
+    { name = "Claude Opus 4 (Latest)", value = "claude-opus-4-20250514" },
+    { name = "Claude Sonnet 4 (Latest)", value = "claude-sonnet-4-20250514" },
+    { name = "Claude 3.7 Sonnet (Latest)", value = "claude-3-7-sonnet-latest" },
+    { name = "Claude 3.5 Haiku (Latest)", value = "claude-3-5-haiku-latest" },
+  }
```

Comment:
Do you think we can make this a user-configurable table?
This could be included in the defaults, and then when a new model version is released, one can update their configuration.